PyInstaller/depend/dylib.py

```diff
@@ -179,6 +195,7 @@ def include_library(libname):
     """
     # For configuration phase we need to have exclude / include lists None
     # so these checking is skipped and library gets included.
+    libname = os.path.basename(libname)
```

Comment:
`basename` cannot be used here directly because it will break `include_library()` function for osx and windows where it is necessary to have full paths.  Please use `basename` only at places where direct regex matching is done.